app/models/size.rb

```diff
@@ -1,2 +1,7 @@
 class Size < ApplicationRecord
+
+  VALID_SIZES = %w(S M L XL)
```

Comment:
Of note: client has a few socks available in XXL. Might be one in there with XXXL as well